roles/ovirt-manageiq/README.md

```diff
@@ -0,0 +1,105 @@
+oVirt/RHV ManageIQ/CloudForms deployment
+========================================
+
+This role downloads ManageIQ/CloudForms qcow image and deploys it into oVirt/RHV.
+Create a virtual machine and attach the ManageIQ/CloudForms disk. Then wait for
+initialization of the ManageIQ/CloudForms system and register the RHV/oVirt as
+a infrastructure provider.
+
+Requirements
+------------
+
+oVirt has to be 4.0.4 or higher and [ovirt-imageio](http://www.ovirt.org/develop/release-management/features/storage/image-upload/) must be installed and running.
+The target machine has to have installed [Python SDK](https://pypi.python.org/pypi/ovirt-engine-sdk-python/4.0.4).
+
+Check `ovirt-imageio-proxy` is running on engine:
+
+```
+systemct status ovirt-imageio-proxy
+```
+
+Also check if `ovirt-imageio-daemon` is running on hosts:
+
+```
+systemct status ovirt-imageio-daemon
+```
+
+You also have to use your CA certificate of the engine.
+You must configure the `ovirt_ca` varible with the path to the CA.
+
+Role Variables
+--------------
+
+QCOW varibles:
+```
+ - miq_qcow_url - Define the URL where the ManageIQ/CloudForms qcow is stored (default: http://releases.manageiq.org/manageiq-ovirt-euwe-1.ova).
+```
+
+oVirt/RHV engine login variables:
+| Name               | Default value     |  Description                                    |
+|--------------------|-------------------|-------------------------------------------------|
+| ovirt_user         | UNDEF             | Define the user to be used to access oVirt/RHV. |
+| ovirt_password     | UNDEF             | Define password of the 'ovirt_user'.            |
+| ovirt_fqdn         | UNDEF             | Define the FQDN of the oVirt/RHV.               |
+| ovirt_ca           | UNDEF             | Define the path to the CA of the oVirt/RHV.     |
+
+Virtual Machine variables:
+| Name               | Default value     |  Description                                 |
+|--------------------|-------------------|----------------------------------------------|
+|  miq_vm_name       | manageiq_euwe     | Define the name of the ManageIQ/CloudForms VM in oVirt/RHV.|
+| miq_vm_cluster     | Default           | Define the cluster of the virtual machine    |
+| miq_vm_memory      | 6GiB              | Define the system memory of the virtual machine    |
+| miq_vm_cpu         | 2                 | Define the number of virtual machine CPU cores   |
+| miq_vm_os          | rhel_7x64         | Define the operating system of the virtual machine |
```

Comment:
Ahh, so this is oVirt specific, we need to set VM OS to RHEL, even though CentOS is used for ManageIQ VM.